src/components/Register/AccountDetails/SaveAccount/SaveAccount.js

```diff
@@ -63,12 +98,37 @@ export default class SaveAccount extends React.Component {
     }
   }
 
-  save = async (filename) => {
+
+  loadWallet = (filename) => {
+    const { alert } = this.props;
+
+    try {
+      const walletLoaded = wallet.Wallet.readFile(filename);
+      return walletLoaded;
+    } catch (err) {
+      alert(`Error loading wallet file: ${err.message}`);
+      return null;
+    }
+  }
+
+
+  save = async (filename, walletLoaded = null) => {
     const { label, account } = this.props;
-    const data = JSON.stringify(accountToNEP6({ ...account, label }));
+    let data = null;
+
+    if (walletLoaded == null) {
```

Comment:
This conditional seems odd for this function.  I would suggest changing the existing `this.save(...)` call from `handleSaveNewWallet` to instead pass in `walletLoaded` with its representative wallet instance.